src/gfp_test.cpp

```diff
@@ -0,0 +1,86 @@
+#include <constants.h>
```

Comment:
Tests should be eventually integrated into CTEST so that they can be run automatically in CICD.

More corner cases should be added in the future.

Reply:
Done, commit: https://github.com/AntelopeIO/bn256/pull/1/commits/f3870d5ee4aa9a93d08f1e0fb83c870b68ff76c4